### PR TITLE
Add support for miatoll (Xiaomi Redmi Note 9 Pro / 9S ... Poco M2 Pro)

### PR DESCRIPTION
**Pull request by anon1892**
Related to issue #85. Not tested !

Requirements : 
- #218 for OrangeFox support (especially `fastboot_flash_recovery` and `fastboot_reboot_recovery`
- #221 for device specific notes (optional)

Actually, the best way of testing may be using the `main` branch of [my fork](https://github.com/anon1892/oai/tree/main), since it need 3 PR to work as is.

### CODE REVIEW

File: openandroidinstaller/assets/configs/miatoll.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  supported_recovery:
+    - orangefox
+    - twrp
+  supported_device_codes:
+    - miatoll
+    - gram
+    - curtana
+    - excalibur
+    - joyeuse
+  notes: >
```
Comment (MagicLike) [inline openandroidinstaller/assets/configs/miatoll.yaml:15]: Shouldn't this be changed to `brand: xiaomi` now?

**Comment by MagicLike** [comment]
Tagging @RaptaG

File: openandroidinstaller/assets/configs/miatoll.yaml (hunk 1 above)
Comment (anon1892) [reply openandroidinstaller/assets/configs/miatoll.yaml]: Yes, done for all devices.

**Comment by MagicLike** [comment]
> How do it test it?

Two ways: You temporaily build OAI locally (this will include using the terminal) or I / @anon1892 can try to build you a version.

**Comment by anon1892** [comment]
@RaptaG As written in the first post of this pull request, the simplest way is to clone the `main` branch of [my repo](https://github.com/anon1892/oai/tree/main)
Then follow the README to start app. (`make poetry`, `make install` and `make app`)

**Comment by MagicLike** [comment]
> @RaptaG As written in the first post of this pull request, the simplest way is to clone the `main` branch of [my repo](https://github.com/anon1892/oai/tree/main) Then follow the README to start app. (`make poetry`, `make install` and `make app`)

Oh I overread that - do you just merge all your commits to the branches into your main?

**Comment by anon1892** [comment]
Not the last ones (changes in `notes`, `brand` and `untested`), but should work the same way.

**Comment by MagicLike** [comment]
> I just tested building, it just builds Linux executable, not windows one. Don't know how to do sweat_smile

Are you on Linux?

**Comment by anon1892** [comment]
Yes

**Comment by MagicLike** [comment]
> Yes

Ohhh ok, I think Windows needs to be build on Windows... But don't worry, just head to the "Actions" tab of your repo and enable actions. Then run `manual-build`. It should spit out Linux, Mac and Windows files

**Comment by MagicLike** [comment]
@RaptaG On what OS is your PC?

**Comment by anon1892** [comment]
Thank you for the explanation.
For Windows, it's here : https://github.com/anon1892/oai/suites/14768652533/artifacts/839864735
For MacOS : https://github.com/anon1892/oai/suites/14768652533/artifacts/839864730
For Linux : https://github.com/anon1892/oai/suites/14768652533/artifacts/839864734

**Comment by RaptaG** [comment]
> @RaptaG On what OS is your PC?

I run Fedora Linux, yes I could compile the app locally

**Comment by MagicLike** [comment]
> > @RaptaG On what OS is your PC?
> 
> I run Fedora Linux, yes I could compile the app locally

no need, it is already compiled and should work -> https://github.com/anon1892/oai/suites/14768652533/artifacts/839864734

**Comment by anon1892** [comment]
I updated the links :wink:

**Comment by anon1892** [comment]
Ah ? I tried in private tab (to know if they was session only), and was redirected to 404

**Comment by MagicLike** [comment]
> Ah ? I tried in private tab (to know if they was session only), and was redirected to 404

GitHub Actions artifacts can only be downloaded with an account

**Comment by RaptaG** [comment]
> GitHub Actions artifacts can only be downloaded with an account

you could also use https://nightly.link

**Comment by anon1892** [comment]
Ok. Good to know, thanks both.

**Comment by RaptaG** [comment]
🎉

**Pull request merged**